luigi_pipeline/lib/hail_vep_runners.py

```diff
@@ -0,0 +1,557 @@
+from abc import ABC, abstractmethod
+
+import hail as hl
+
+from hail_scripts.v02.utils import hail_utils
+
+
+class HailVEPRunnerBase(ABC):
+
+    @abstractmethod
+    def run(self, mt, genome_version):
+        pass
+
+
+class HailVEPRunner(HailVEPRunnerBase):
+
+    def run(self, mt, genome_version):
+        return hail_utils.run_vep(mt, genome_version)
+
+
+class HailVEPDummyRunner(HailVEPRunnerBase):
+
+    MOCK_VEP_DATA = hl.struct(
```

Comment:
This is copied from one of the vep annotations in the 1kg variants. Ideally, this would be generic data and labeld as `mock` somehow, but it involves a lot of genetics domain knowledge that I don't know. I'll add to comments.